config/initializers/batch_form_terms.rb

```diff
@@ -0,0 +1,4 @@
+# This sets up the correct terms for the batch works forms
```

Comment:
I'm guessing setting these in an initializer is a Hyrax convention?

Reply:
Dunno, I'm not sure how else you would setup the correct terms for that form. 

https://github.com/samvera/hyrax/blob/master/app/forms/hyrax/forms/batch_upload_form.rb

inherits from from work_form which has a set of terms that we don't want:

https://github.com/samvera/hyrax/blob/master/app/forms/hyrax/forms/work_form.rb#L24